apps/prairielearn/src/tests/exampleCourseQuestionsComplete.test.ts

```diff
@@ -177,11 +177,25 @@ const validateHtml = async (html: string) => {
       'element-permitted-content': 'off',
     },
   });
+
+  const filteredResults = results.map((result) => {
+    result.messages = result.messages.filter((m) => {
+      if (m.ruleId === 'aria-label-misuse' && m.selector && /^.*> option[^>]*$/.test(m.selector)) {
```

Comment:
Match: `div:nth-child(10) > div > div > div > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > select > option:nth-child(3) `

Reply:
Did you consider doing this with `rewriteValidatorFalsePositives` instead? That's how we've addressed similar false positives before.